sample/.env.staging

```
@@ -0,0 +1,2 @@
+REST_API_ENDPOINT=https://staging.example.com
+SECRET=This is a sample staging secret
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)